.github/workflows/comment-changed-filenames.yml

```
@@ -0,0 +1,79 @@
+name: Flag changed filenames
+
+# **What it does**: Adds or removes a comment if a PR renames or removes a file.
+# **Why we have it**: Highlights when we need redirects covering certain file paths.
+# **Who does it impact**: PCX team, other Cloudflare contributors
+
+on:
+  pull_request:
+    branches:
+      - production
+
+jobs:
+  
+  flag_changed_filenames:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Get Renamed or Removed Files from Pull Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          files=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
```

```suggestion
          files=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
```
Let's use the more common auth header.